test/nodes/test_connector.py

```diff
@@ -16,14 +19,16 @@ def test_url():
     return f"file://{SAMPLES_PATH.absolute()}/crawler"
 
 
-def content_match(crawler: Crawler, url: str, crawled_page: Path):
+def content_match(crawler: Crawler, url: str, crawled_page: Path, loading_wait_time: int = None):
```

Comment:
I'm not sure we need the waiting time parameter here. After all, you added a `page_dynamic_result.html` file that already corresponds to the final version after the wait. Can't we simply use that, without waiting?